v3/src/components/graph/adornments/univariate-measures/box-plot/box-plot-adornment-component.scss

```diff
@@ -76,9 +76,10 @@
   top: 0;
   width: 100%;
 
-  &.box-plot-label {
+  .box-plot-label {
     background: #ffc;
     box-shadow: 1px 1px 1px rgba(51, 51, 51, .7);
     padding: 2px 4px;
+    pointer-events: none;
   }
```

Comment:
I added a commit which puts this back inside the `.measure-labels` block (without the ampersand) which allowed me to remove the `!important`.